README.markdown

```diff
@@ -13,7 +13,7 @@ Head on over to https://facebook.github.io/watchman/
 
 ## Build Status
 
-](https://travis-ci.org/facebook/watchman)
+[![Build Status](https://travis-ci.org/facebook/watchman.svg?branch=master)](https://travis-ci.org/facebook/watchman)
```

Comment:
we don't use travis any more; it looks like I typo'd an update to this readme; would you mind adjusting this PR to just delete this line?

Reply:
Sure!